From dec8b451162ebd975ae854b3d45babc125111248 Mon Sep 17 00:00:00 2001
From: Eric Naim <dnaim@cachyos.org>
Date: Wed, 25 Mar 2026 03:53:14 +0800
Subject: [PATCH 004/243] mm: Increase VM_READAHEAD_PAGES to 256K

Double VM_READAHEAD_PAGES from the default 128K to increase throughput
but not so much as it harms latency.

Signed-off-by: Eric Naim <dnaim@cachyos.org>
---
 include/linux/pagemap.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/pagemap.h b/include/linux/pagemap.h
index 2c3718d592d6..42dba9e2da89 100644
--- a/include/linux/pagemap.h
+++ b/include/linux/pagemap.h
@@ -1363,7 +1363,7 @@ struct readahead_control {
 		._index = i,						\
 	}
 
-#define VM_READAHEAD_PAGES	(SZ_128K / PAGE_SIZE)
+#define VM_READAHEAD_PAGES	(SZ_256K / PAGE_SIZE)
 
 void page_cache_ra_unbounded(struct readahead_control *,
 		unsigned long nr_to_read, unsigned long lookahead_count);
-- 
2.55.0

